app/models/maat_api/court_application.rb

```diff
@@ -107,11 +109,11 @@ def offence_wording
     end
 
     def court_centre_short_ou_code
-      find_court_centre_by_id(hearing.court_centre_id).short_oucode
+      find_court_centre_by_id(hearing_resulted.hearing_court_centre_id).short_oucode
     end
 
     def hearing_first_sitting_day_date
-      hearing.first_sitting_day_date&.to_date&.strftime("%Y-%m-%d")
+      hearing_resulted.hearing.first_sitting_day_date&.to_date&.strftime("%Y-%m-%d")
```

Comment:
Why not `hearing_resulted.hearing_first_sitting_day_date...` with delegate and prefex for `first_sitting_day_date`?

Reply:
So it follows the same pattern as jurisdiction_type and court_centre_id

Reply:
Yeah, good idea 👌🏻